Tiltfile

```diff
@@ -218,6 +218,8 @@ def create_crs():
     local(kubectl_cmd + " delete configmaps calico-ipv6-addon --ignore-not-found=true")
     local(kubectl_cmd + " create configmap calico-ipv6-addon --from-file=templates/addons/calico-ipv6.yaml")
     local(kubectl_cmd + " delete configmaps flannel-windows-addon --ignore-not-found=true")
```

Comment:
unrelated to this PR but @jsturtevant do you know why we are deleting the flannel-windows-addon cm but not creating it?

Reply:
it's created on line https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2012/files#diff-c2ee8653e1d6b85f0aadf87cd438a9250806c052877248442be4d434cbc52425R226

thought it doesn't look like we delete `calico-windows-addon` which is added the line after.  Not sure if deletion is required... Seems to be working, even on reboots of tilt

Reply:
I was going to open another PR to fix that